tools/build/third_party/perfetto/perfetto.BUILD

```diff
@@ -218,8 +224,14 @@ cc_library(
 )
 
 cc_library(
-    name = "tracing_core",
+    name = "tracing_ipc",
     srcs = [
+        "src/base/watchdog_posix.cc",
```

Comment:
remove, this should not be necessary anymore and should only be used by standalone builds.

Reply:
This is needed if we try and link perfetto to the vulkan layer.